Tests/PubNubTests/Networking/Routers/PushRouterTests.swift

```diff
@@ -133,50 +124,34 @@ final class PushEndpointTests: XCTestCase {
         case .success:
           XCTFail("This should not succeed")
         case let .failure(error):
-          XCTAssertEqual(error.pubNubError, PubNubError(reason: .pushNotEnabled))
+          XCTAssertEqual(error.pubNubError, PubNubError(.pushNotEnabled))
         }
         expectation.fulfill()
       }
 
     wait(for: [expectation], timeout: 1.0)
   }
+}
 
-  // MARK: - Modify Push Channels Tests
+// MARK: - Modify Push Channels Tests
 
-  func testModifyPushChannels_Endpoint() {
+extension PushRouterTests {
+  func testModifyPushChannels_Router() {
     guard let data = Data(hexEncodedString: "A1B2") else {
       return XCTFail("Could not encode Data from hex string")
     }
-    let endpoint = Endpoint.modifyPushChannels(pushToken: data,
-                                               pushType: .apns,
-                                               addChannels: testChannels,
-                                               removeChannels: [])
-
-    XCTAssertEqual(endpoint.description, "Modify Push Channels")
-    XCTAssertEqual(endpoint.category, .modifyPushChannels)
-    XCTAssertEqual(endpoint.operationCategory, .push)
-    XCTAssertNil(endpoint.validationError)
-  }
 
-  func testListModifyPushChannels_Endpoint_ValidationError() {
-    let endpoint = Endpoint.modifyPushChannels(pushToken: Data(),
-                                               pushType: .apns,
-                                               addChannels: [],
-                                               removeChannels: [])
+    let router = PushRouter(.modifyPushChannels(pushToken: data, pushType: .apns, joining: testChannels, leaving: []), configuration: config)
 
-    XCTAssertNotEqual(endpoint.validationError?.pubNubError, PubNubError(.invalidEndpointType, endpoint: endpoint))
+    XCTAssertEqual(router.endpoint.description, "Modify Push Channels")
+    XCTAssertEqual(router.category, "Modify Push Channels")
+    XCTAssertEqual(router.service, .push)
   }
 
-  func testListModifyPushChannels_Endpoint_AssociatedValues() {
-    let endpoint = Endpoint.modifyPushChannels(pushToken: Data(),
-                                               pushType: .apns,
-                                               addChannels: ["SomeChannel"],
-                                               removeChannels: ["OtherChannel"])
+  func testListModifyPushChannels_Router_ValidationError() {
+    let router = PushRouter(.modifyPushChannels(pushToken: Data(), pushType: .apns, joining: [], leaving: []), configuration: config)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 133 characters](https://app.codacy.com/app/PubNub/swift/pullRequest?prid=4513555)